packages/scene-composer/src/components/panels/scene-components/tag-style/ColorPicker/ColorPickerUtils/SvgParserHelper.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+export const replaceFillAttribute = (element: Element, selectedColor: string): void => {
+  if (element === undefined || selectedColor === undefined) {
+    return;
+  }
+  const tagName = element.tagName.toLowerCase();
+  if (tagName === 'ellipse') {
+    element.setAttribute('stroke', selectedColor);
+  }
+  if (tagName === 'circle') {
+    element.setAttribute('fill', selectedColor!);
+  }
```

Comment:
When a new `tagName` is supported, does other code need to change?

Reply:
I was more curious if you added a rectangle, for example, is this the only function we need to change?

Reply:
yup. We need to make it more generic by adding appropriate checks based on the svg.